build/pipelines/azure-pipelines.release.yaml

```diff
@@ -44,7 +44,7 @@ jobs:
   parameters:
     platform: x86
 
```

Comment:
I suppose we should bring back UI tests to this release pipeline.
The UI tests were removed because window-2019 is not compatible with the min version(*10.0.22000.0*) specified in the [release manifest](https://github.com/microsoft/calculator/blob/aba642351360e980bfe5fa6ee731c82055378036/src/Calculator/Package.Release.appxmanifest#L11).

Reply:
windows-2022 = Windows Server 2022 = 10.0.20348.0, which is still not compatible with 10.0.22000.0.

We'll need to stand up a pool of real Windows 11 desktop images to get the UI tests back.